tests/cortexm_common_ldscript/Makefile

```diff
@@ -4,18 +4,11 @@ include ../Makefile.tests_common
 
 # Normally all boards using `cortexm_common/ldscripts/cortexm.ld` linkerscript
 # Only tested on these ones for the moment
-BOARD_WHITELIST += iotlab-a8-m3
-BOARD_WHITELIST += iotlab-m3
-BOARD_WHITELIST += samr21-xpro
+BOARD_WHITELIST += frdm-kw41z frdm-k64f frdm-k22f iotlab-a8-m3 iotlab-m3 \
```

Comment:
Here I prefered the one board per line version for readability.
Also the kinetis` boards should be more with a comment saying "using `cpu/kinetis/kinetis.ld` as they do not use `cortem.ld` directly.

Reply:
Ok, I was just going with the structure usually used in tests, but I'll change it.